code/modules/mob/living/simple_animal/abnormality/he/you_strong.dm

```diff
@@ -241,17 +241,15 @@
 /mob/living/simple_animal/hostile/grown_strong/Move(atom/newloc, dir, step_x, step_y)
 	if(status_flags & GODMODE)
 		return FALSE
-	. = ..()
-	return
+	return ..()
 
 /mob/living/simple_animal/hostile/grown_strong/AttackingTarget(atom/attacked_target)
 	if(status_flags & GODMODE)
 		return FALSE
-	. = ..()
-	return
+	return ..()
 
 /mob/living/simple_animal/hostile/grown_strong/Initialize()
-	. = ..()
+	return ..() // replace with . = ..() once the icon state code gets un-commented
 	//icon_state = pick("1", "2", "3")
```

Comment:
Just remove it entirely; We don't need unused/commented out code blocks.

Reply:
if you say so, just removed the whole initialize override code since that comment was preety much the only reason it existed